src/script/ceph-release-notes

```diff
@@ -62,6 +63,19 @@ def make_release_notes(gh, repo, ref, plaintext):
                 print ("ERROR: http://github.com/ceph/ceph/pull/" + str(number) + " has no associated issue")
                 continue    
 
+            if original_issue:
+                r = requests.get(tracker_uri.format(issue),params={"include":"relations"})
+                # Now we can get an original issue only if the
+                # relations field is populated, otherwise we fallback
+                # to the tracker number
+                try:
+                    relations = r.json()["issue"]["relations"]
+                except KeyError:
+                    break
```

Comment:
Actually, we it should not be continue either, just make it so the rest of the **if original_issue** is not run